fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/api/os/ServerOperatingSystem.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2020 Fraunhofer FOKUS and others. All rights reserved.
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contact: mosaic@fokus.fraunhofer.de
+ */
+
+package org.eclipse.mosaic.fed.application.app.api.os;
+
+/**
+ * Note: This interface is empty for now and currently only functions as a marker-interface. Future extensions
+ * might add features.
+ */
+public interface ServerOperatingSystem extends OperatingSystem {
```

Comment:
Here it is ok to have an empty interface, as we really need it for a new type of operating system. Without this, we would not be able to create an application.